ArduCopter/ArduCopter.cpp

```diff
@@ -74,87 +74,82 @@
+    AP_Task<Copter>::create(&Copter::ekf_check,             10,     75),
+    AP_Task<Copter>::create(&Copter::landinggear_update,    10,     75),
+    AP_Task<Copter>::create(&Copter::lost_vehicle_check,    10,     50),
+    AP_Task<Copter>::create(&Copter::gcs_check_input,      400,    180),
+    AP_Task<Copter>::create(&Copter::gcs_send_heartbeat,     1,    110),
+    AP_Task<Copter>::create(&Copter::gcs_send_deferred,     50,    550),
+    AP_Task<Copter>::create(&Copter::gcs_data_stream_send,  50,    550),
+    AP_Task<Copter>::create(&Copter::update_mount,          50,     75),
+    AP_Task<Copter>::create(&Copter::update_trigger,        50,     75),
+    AP_Task<Copter>::create(&Copter::ten_hz_logging_loop,   10,    350),
+    AP_Task<Copter>::create(&Copter::twentyfive_hz_logging, 25,    110),
+    AP_Task<Copter>::create(&Copter::dataflash_periodic,    400,    300),
+    AP_Task<Copter>::create(&Copter::perf_update,           0.1,    75),
+    AP_Task<Copter>::create(&Copter::read_receiver_rssi,    10,     75),
+    AP_Task<Copter>::create(&Copter::rpm_update,            10,    200),
+    AP_Task<Copter>::create(&Copter::compass_cal_update,   100,    100),
+    AP_Task<Copter>::create(&Copter::accel_cal_update,      10,    100),
 #if ADSB_ENABLED == ENABLED
-    SCHED_TASK(avoidance_adsb_update, 10,    100),
+    AP_Task<Copter>::create(&Copter::avoidance_adsb_update, 10,    100),
 #endif
 #if ADVANCED_FAILSAFE == ENABLED
-    SCHED_TASK(afs_fs_check,          10,    100),
+    AP_Task<Copter>::create(&Copter::afs_fs_check,          10,    100),
 #endif
-    SCHED_TASK(terrain_update,        10,    100),
+    AP_Task<Copter>::create(&Copter::terrain_update,        10,    100),
 #if GRIPPER_ENABLED == ENABLED
-    SCHED_TASK(gripper_update,        10,     75),
+    AP_Task<Copter>::create(&Copter::gripper_update,        10,     75),
 #endif
 #ifdef USERHOOK_FASTLOOP
-    SCHED_TASK(userhook_FastLoop,    100,     75),
+    AP_Task<Copter>::create(&Copter::userhook_FastLoop,    100,     75),
 #endif
 #ifdef USERHOOK_50HZLOOP
-    SCHED_TASK(userhook_50Hz,         50,     75),
+    AP_Task<Copter>::create(&Copter::userhook_50Hz,         50,     75),
 #endif
 #ifdef USERHOOK_MEDIUMLOOP
-    SCHED_TASK(userhook_MediumLoop,   10,     75),
+    AP_Task<Copter>::create(&Copter::userhook_MediumLoop,   10,     75),
 #endif
 #ifdef USERHOOK_SLOWLOOP
-    SCHED_TASK(userhook_SlowLoop,     3.3,    75),
+    AP_Task<Copter>::create(&Copter::userhook_SlowLoop,     3.3,    75),
 #endif
 #ifdef USERHOOK_SUPERSLOWLOOP
-    SCHED_TASK(userhook_SuperSlowLoop, 1,   75),
+    AP_Task<Copter>::create(&Copter::userhook_SuperSlowLoop, 1,   75),
 #endif
-    SCHED_TASK(button_update,          5,    100),
-    SCHED_TASK(stats_update,           1,    100),
+    AP_Task<Copter>::create(&Copter::button_update,          5,    100),
+    AP_Task<Copter>::create(&Copter::stats_update,           1,    100),
```

Comment:
To avoid missing items, instead of just rebasing I would redo the changes on what we have today. It's just a search and replace: `s/SCHED_TASK(/AP_Task<Copter>::create(&Copter::/g`. Otherwise, make sure all items are there.